Docs issues on Production (#4690)
mattermost/docs: source/deployment/bulk-loading-data.rst

  ``sudo -u mattermost bin/mattermost import bulk data.jsonl --apply``

5. When the bulk load command completes, clear all caches by going to **System Console > Environment > Web Server**.

.. important::

>   Owner for ``data`` directory and all its content has to change to Mattermost user for it to work correctly. Otherwise, Mattermost cannot fetch the files created in ``data`` dir after the import tool has run, since all files created in ``data`` dir are owned by ``root`` as the tool was run as ``sudo``.

Reviewer: ```suggestion
After the import tool has run, all files created in the ``data`` directory are owned by *root* as the tool was run as *sudo*. The owner of the ``data`` directory and all its content has to change to *mattermost* user, otherwise, Mattermost can't fetch the files created in the ``data`` directory after the import tool has run.
```

Reviewer: Rewording for clarity.

Author: Thank you so much -- this is much clearer.
